front-end/.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Might want to add the .idea directory to the .gitignore. Otherwise you guys are probably going to muck around with people's project settings.

Reply:
@Richard-outreach Updated. Thanks for the heads up!